components/solutions/comparisonPage.tsx

```diff
@@ -49,8 +49,8 @@ export default function ComparisonPage(comp: ComparisonInfo) {
 
           <Row>
             <Col>
-              <Button variant="primary" href="/meet" className="ml-auto">
-                Get a Demo →
+              <Button variant="primary" href="/get-started" className="ml-auto">
+                Start Free Trial
```

Comment:
I imagine the same should be changed on the `/solutions/*` pages too? They still point to `/meet`